communities.md

```diff
@@ -29,9 +29,11 @@ Project-specific, but likely of interest to the general C++ community.
 		- usage: irc://irc.freenode.org/gcc - http://irc.netsplit.de/channels/details.php?room=%23gcc&net=freenode
 		- development: irc://irc.oftc.net/gcc - http://irc.netsplit.de/channels/details.php?room=%23gcc&net=OFTC
 	+ Mailing Lists - https://gcc.gnu.org/lists.html
-* LLVM
+* LLVM                    
 	+ IRC - https://llvm.org/docs/#irc - irc://irc.oftc.net/llvm
+	+ Discord (Chat) - https://discord.gg/xS7Z362
 	+ Mailing Lists - http://lists.llvm.org/mailman/listinfo
+	+ Discourse (Discussion Forums) - https://llvm.discourse.group/
```

Comment:
Likely reasonable to just follow the order on llvm.org ("Useful Links"): Mailing Lists, Discourse, IRC Channel, Discord